scripts/update_setup_py.sh

```diff
@@ -43,7 +43,7 @@ echo -e "rules:
         constraint_files = set()
 
         # groups \"my-package-name<=x.y.z,...\" into (\"my-package-name\", \"<=x.y.z,...\")
-        requirement_line_regex = re.compile(r\"([a-zA-Z0-9-_.]+)([<>=][^#\s]+)?\")
+        requirement_line_regex = re.compile(r\"([a-zA-Z0-9-_.\[\]]+)([<>=][^#\s]+)?\")
```

Comment:
```suggestion
        requirement_line_regex = re.compile(r\"([a-zA-Z0-9\-_.\[\],]+)([<>=][^#\s]+)?\")
```
There can be more than one extra in the list, so we should have a comma as well. And probably want to escape this hyphen, even though it *shouldn't* cause trouble at the moment (since there's a range just before it).

Reply:
Theoretically, there could also be whitespace around that comma or probably even the brackets. I don't see any instances of that in our code, though. :thinking: